### ISSUE TITLE
No server TLS configuration available

### ISSUE THREAD

**Issue by queicherius** [getmeli/meli#161]
I ran into another issue trying to set up Meli via the newest Docker image. I keep getting the following error:

```
{"level":"debug","ts":1607999698.0814688,"logger":"http.stdlib","msg":"http: TLS handshake error from <my ip redacted>:63030: no server TLS configuration available for ClientHello: &{CipherSuites:[31354 4865 4866 4867 49195 49199 49
196 49200 52393 52392 49171 49172 156 157 47 53] ServerName:palette.meli.devoxa.io SupportedCurves:[CurveID(19018) X2551
9 CurveP256 CurveP384] SupportedPoints:[0] SignatureSchemes:[ECDSAWithP256AndSHA256 PSSWithSHA256 PKCS1WithSHA256 ECDSAW
ithP384AndSHA384 PSSWithSHA384 PKCS1WithSHA384 PSSWithSHA512 PKCS1WithSHA512] SupportedProtos:[h2 http/1.1] SupportedVer
sions:[14906 772 771 770 769] Conn:0xc00085a8c8 config:0xc000537800}"}
```

It looks like this might be the cause, but I am not sure https://github.com/caddyserver/caddy/issues/3004 - it did work just fine with the version of a couple of days ago. I have since set up Meli from Scratch and it has not fixed the issue.

**Comment by gempain** [comment]
@queicherius many thanks for reporting and greatest apologies for the inconvenience, we're on it. The major change in the previous release is that we moved all routes of a site so a single subroute group.

Will get back to you ASAP and release a fix today.

**Comment by gempain** [comment]
This has been fixed in v1.0.0-beta.6. Sorry for the delay, we're on CET 😀 

I'm closing, but you may continue replying if further help is required. We'l reopen if needed. Again, many thanks for reporting this, greatly appreciate you taking the time to contribute.

**Comment by queicherius** [comment]
> This has been fixed in v1.0.0-beta.6.

Awesome, that works like a charm! 

> Sorry for the delay, we're on CET 😀

You are already extremely fast, no need to apologize 💯
